lib/commands/push.ts

```diff
@@ -224,6 +225,18 @@ export default class PushCmd extends Command {
 			char: 'g',
 			exclusive: ['multi-dockerignore'],
 		}),
+		'release-tag': flags.string({
+			description: stripIndent`
+				When performing a push to an application, use --release-tag in order set the
+				release tags and their values of your choice.
+
+				e.g:
+					balena push MyApp --release-tag key1 value1 key2 "value2 with spaces" key3 "" key4
+
+				Note that in order for the tags to be applied the push operation should be successful.
+			`,
```

Comment:
I think the sample command line fits better in the Examples section of the documentation:
https://github.com/balena-io/balena-cli/blob/v12.35.2/lib/commands/push.ts#L91-L103

`balena push` is an outlier in terms of the length of the documentation of each flag. The documentation of most flags of most other commands is a single line. We can have longer documentation where it is needed, no problem, but we should still aim at being concise. Suggestion:
```
	'release-tag': flags.string({
		description: stripIndent`
			Set release tags if the push to a cloud application is successful. Multiple
			arguments may be provided, alternating tag keys and values (see examples).`,
		multiple: true,
	}),
```